source/reporting/index.html.md.erb

```diff
@@ -111,7 +117,14 @@ For example:
 }
 ```
 
-You should receive the payment in your bank account within 2 business days of `captured_date`.
+If your PSP is Stripe, you should receive the payment in your bank account within:
+
+- 2 working days of `captured_date` if your user completed their payment during the week
+- 3 working days of `captured_date` if your user completed their payment at the weekend or on a bank holiday
+
+You can confirm when payments are paid into your bank account by signing in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) and going to __My Services__ > __View payments to your bank account__.
+
+If your PSP is Government Banking’s contracted PSP (currently Worldpay), SmartPay or ePDQ, you should receive the payment in your bank account within 2 working days of `captured_date`.
```

Comment:
Should match the text we have in the Integrate with the API section

If your PSP is Government Banking’s contracted PSP (currently Worldpay), SmartPay or ePDQ, contact your PSP to find out your minimum payout and payment times.